src/main/scala/dissonance/Discord.scala

```diff
@@ -128,8 +134,15 @@ class Discord(token: String, httpClient: Client[IO], wsClient: WSClient[IO])(imp
   private def makeHeartbeat(sequenceNumber: SequenceNumber) =
     sequenceNumber.get.map(Heartbeat.apply).map(heartbeat => Text(heartbeat.asJson.noSpaces))
 
-  private def identityMessage =
-    Text(s"""{"op":2,"d":{"token":"$token","intents":32509,"properties":{"$$os":"","$$browser":"","$$device":""}}}""")
+  private def identifyMessage(intents: List[Intent]) = // TODO: Make a case class for the op, d structure that is also used in ControlMessage
+    Text(
+      Json
+        .obj(
+          "op" -> 2.asJson,
+          "d"  -> Identify(token, IdentifyConnectionProperties("", "", ""), None, None, None, None, None, intents).asJson
+        )
+        .spaces2SortKeys
```

Comment:
Since it's not displayed and just going over the network maybe this:
```suggestion
        .noSpaces
```